<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QMesh</class>
 <widget class="QWidget" name="QMesh">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>651</width>
    <height>377</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="10,3">
   <property name="sizeConstraint">
    <enum>QLayout::SetDefaultConstraint</enum>
   </property>
   <item>
    <layout class="QVBoxLayout" name="plotLayout"/>
   </item>
   <item>
    <layout class="QVBoxLayout" name="userLayout" stretch="0,0,0,0,0">
     <property name="spacing">
      <number>5</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="addElementLayout">
       <item>
        <widget class="QPushButton" name="addElementButton">
         <property name="text">
          <string>Добавить элемент</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cancelElementButton">
         <property name="text">
          <string>Отмена</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="addWidgetsLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetMinimumSize</enum>
       </property>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="itemListLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Список элементов:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableView" name="itemList">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>1</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="createMeshButton">
       <property name="text">
        <string>Создать сетку</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
